src/endpoints/transactions/transaction.get.service.ts

```diff
@@ -116,6 +116,14 @@ export class TransactionGetService {
         }
       }
 
+      if (transaction.senderUserName) {
+        transactionDetailed.senderUsername = UsernameUtils.extractUsernameFromRawBase64(transaction.senderUserName);
```

Comment:
let's prepare for the upcoming ES upgrade and add support for both `senderUserName` and `senderUsername` fields directly

Reply:
done